packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -90,6 +97,13 @@ class AccessLegBody extends Component<Props, State> {
       );
     }
 
+    const Mapillary = () => (
```

Comment:
Just make this a variable.
```js
const mapillary = (
  <MapillaryButton ... />
)
```

Reply:
Then write:
```js
<SomeContainer>
  {mapillary}
</SomeContainer>
```

Reply:
Added in [c0931b3](https://github.com/opentripplanner/otp-ui/pull/658/commits/c0931b308342f6fd92ac7dfc8192b44a20dda5c9)!